File: ws/Perez/U2/WS24/Zoo/src/main/java/ec/edu/espe/zoo/model/Feline.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Carlos Perez,The Javas, DCCO-ESPE
 */
public abstract class Feline extends Mammal{
    
    public Feline(int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, boolean vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
    }
    
}

File: hw/Pérez/U2/HW18ZooJavaScript/model/Feline.js
import { Mammal } from './Mammal.js';

export class Feline extends Mammal {
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
    }
}
